src/test/java/org/openmrs/contrib/qaframework/automation/AttachmentSteps.java

```diff
@@ -0,0 +1,117 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openmrs.contrib.qaframework.page.AttachmentsPage;
+import org.openmrs.reference.page.HomePage;
+
+import io.cucumber.java.After;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class AttachmentSteps extends Steps {
+	
+	private AttachmentsPage attachmentsPage;
+	
+	@After("@selenium")
+	public void destroy() {
+		quit();
+	}
+	
+	@Given("a user logs in, searches a patient and visits Attachment page")
+	public void loadPatientDashboard() {
+		goToLoginPage().login(testProperties.getUsername(), testProperties.getPassword(), testProperties.getLocation());
+		homePage = new HomePage(loginPage);
+		findPatientPage = homePage.goToFindPatientRecord();
+		findPatientPage.clickOnFirstPatient();
+		dashboardPage = findPatientPage.clickOnFirstPatient();
+	}
+	
+	@And("a user clicks on Attachment link")
+	public void userClicksOnAttachmentLink() {
+		attachmentsPage = new AttachmentsPage(dashboardPage);
+		attachmentsPage.clickOnAttachmentLink();
+	}
+	
+	@Then("the system loads Attachment page")
+	public void systemLoadsAttachmentPage() {
+		assertEquals(getElement(patientHeaderId).getText(), getElement(patientHeaderId).getText());
+		assertTrue(textExists("Attachments"));
+	}
+	
+	@And("a user attaches a file")
+	public void userAttachesFile() {
+		attachmentsPage.clickOrDropFile();
```

Comment:
clickOrDropFile may be containing some slight typo!

Reply:
Oops! It was an oversight, however,  I caught the typo after taking a look at it carefully